src/server/scripts/Events/pilgrims_bounty.cpp

```diff
@@ -532,16 +540,16 @@ class spell_pilgrims_bounty_turkey_tracker : public SpellScript
                 switch (stackAmount)
                 {
                     case 10:
-                        target->TextEmote("Turkey Hunter!", target, true);
+                        target->Talk(EMOTE_TURKEY_HUNTER, CHAT_MSG_RAID_BOSS_WHISPER, sWorld->getFloatConfig(CONFIG_LISTEN_RANGE_TEXTEMOTE), target);
                         break;
                     case 20:
-                        target->TextEmote("Turkey Domination!", target, true);
+                        target->Talk(EMOTE_TURKEY_DOMINATION, CHAT_MSG_RAID_BOSS_WHISPER, sWorld->getFloatConfig(CONFIG_LISTEN_RANGE_TEXTEMOTE), target);
                         break;
                     case 30:
-                        target->TextEmote("Turkey Slaughter!", target, true);
+                        target->Talk(EMOTE_TURKEY_SLAUGHTER, CHAT_MSG_RAID_BOSS_WHISPER, sWorld->getFloatConfig(CONFIG_LISTEN_RANGE_TEXTEMOTE), target);
                         break;
                     case 40:
-                        target->TextEmote("TURKEY TRIUMPH!", target, true);
+                        target->Talk(EMOTE_TURKEY_TRIUMPH, CHAT_MSG_RAID_BOSS_WHISPER, sWorld->getFloatConfig(CONFIG_LISTEN_RANGE_TEXTEMOTE), target);
```

Comment:
https://github.com/azerothcore/azerothcore-wotlk/pull/18128/

This would need you to update to WHISPER_TURKEY_... because it is actually a whisper. My bad for not checking with the sniff first x)

Can you test these changes with the PR I linked?